design/api.go

```diff
@@ -21,7 +21,7 @@ var _ = a.API("alm", func() {
 	})
 	a.Origin("/[.*almighty.io|localhost]/", func() {
```

Comment:
I wonder if this needs to be changed from `.*almighty.io` to the new domain. @aslakknutsen ?